<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#newPhotoBtn {
	background-color: rgb(0, 85, 255);	
	border: none;
	background-repeat: none;
	color: white;
	border-radius: 15px;
}

#logoutBtn {
	background-color: rgb(220, 0, 0);
	border: none;
	background-repeat: none;
	color: white;
	border-radius: 15px;
}


#saveBtn {
	background-color: rgb(0, 85, 255);	
	border: none;
	background-repeat: none;
	color: white;
	border-radius: 15px;
}

#cancelBtn {
	background-color: rgb(220, 0, 0);
	border: none;
	background-repeat: none;
	color: white;
	border-radius: 15px;
}
</string>
  </property>
  <widget class="QPushButton" name="newPhotoBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>500</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(109, 169, 225)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>520</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(109, 169, 225)</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>62</y>
     <width>791</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>500</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="BtnColor" stdset="0">
    <color>
     <red>90</red>
     <green>90</green>
     <blue>180</blue>
    </color>
   </property>
  </widget>
  <widget class="QLabel" name="countdownLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>121</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>172</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <zorder>imgLabel</zorder>
  <zorder>newPhotoBtn</zorder>
  <zorder>logoutBtn</zorder>
  <zorder>countdownLabel</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>saveBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
